src/Blocks/components/post-card/post-card-style.scss

```diff
@@ -0,0 +1,42 @@
+.post-card {
+	display: flex;
+	flex-direction: column;
+	height: 100%;
+
+	&__image {
+		padding-bottom: 20px;
+		height: 100%;
+		width: 100%;
+		object-fit: cover;
+	}
+
+	&__image-wrap {
+		height: 60%;
+	}
+
+	&__date {
+		margin-bottom: 2px;
+	}
+
+	&__heading {
+		margin-bottom: 10px;
+
+		display: -webkit-box;
+		-webkit-line-clamp: 2;
+		-webkit-box-orient: vertical;
```

Comment:
we have autoprefixer for this so there is no need to write vendor specific css

Reply:
This is an odd case where every browser vendor made it work only with `-webkit` prefix.
I changed it later to something more standard, though.

Reply:
if you have something in css that is only for one vendor and it is not supported in other, you don't use it.